test/liveValidatorTests.js

```diff
@@ -117,7 +117,7 @@ describe('Live Validator', function () {
       let validator = new LiveValidator(options);
       validator.initialize().then(function () {
         assert.equal(true, expectedProvider in validator.cache);
-        assert.equal(5, Object.keys(validator.cache).length);
+        assert.equal(6, Object.keys(validator.cache).length);
```

Comment:
QQ: Is this because of the test.json or the fix ?

Reply:
yes. I could have added this in a separate folder. But then thought of keeping all the swaggers together. In that way we know that live validator will work well for the newly added edge case scenario tests as well.